<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabelaObjetosWidget</class>
 <widget class="QWidget" name="TabelaObjetosWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="tabelaobj_grid">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>97</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="adicionar_tb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Adicionar Item</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/adicionar.png</normaloff>:/icones/icones/adicionar.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remover_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remover Item</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/remover.png</normaloff>:/icones/icones/remover.png</iconset>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="atualizar_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Atualizar Item</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/atualizar.png</normaloff>:/icones/icones/atualizar.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remover_todas_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Excluir Todos</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/limpartexto.png</normaloff>:/icones/icones/limpartexto.png</iconset>
       </property>
       <property name="shortcut">
        <string>Shift+Del</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editar_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Editar Item</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/editar.png</normaloff>:/icones/icones/editar.png</iconset>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mover_cima_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Mover para cima</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/movercima.png</normaloff>:/icones/icones/movercima.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mover_baixo_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Mover para baixo</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/moverbaixo.png</normaloff>:/icones/icones/moverbaixo.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mover_primeiro_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Mover para o in√≠cio</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/moverprimeiro.png</normaloff>:/icones/icones/moverprimeiro.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mover_ultimo_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Mover para o fim</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/moverultimo.png</normaloff>:/icones/icones/moverultimo.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+End, Ctrl+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>92</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="tabela_tbw">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>15</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
